src/reynir_correct/pattern.py

```diff
@@ -1681,16 +1672,16 @@ def create_patterns(cls) -> None:
                     None,
                 )
             )
-
-            # Catch "Hún á (ekki) heiðurinn að þessu.", "Hún hafði (ekki) átt heiðurinn að þessu."
+            # Catch "Ég lét (ekki) gott að mér leiða."
             p.append(
                 (
-                    "heiður",  # Trigger lemma for this pattern
-                    "VP > { VP >> { VP > { 'eiga' } NP > { 'heiður' } } PP > { 'að' } }",
-                    cls.wrong_preposition_heiður_að,
+                    "leiður",  # Trigger lemma for this pattern
```

Comment:
"leiða" (verb) should be a trigger lemma, that is the correct parse.

Reply:
Done!